crates/connector_proxy/src/interceptors/network_tunnel_capture_interceptor.rs

```diff
@@ -22,7 +22,9 @@ impl NetworkTunnelCaptureInterceptor {
                 RawValue::from_string(request.endpoint_spec_json)?,
             )
             .await
-            .or_bail()
+            .map_err(|err| {
+                create_custom_error(&format!("error consuming tunnel configuration {:?}", err))
```

Comment:
supernit: you can interpolate `err` directly into the `format!` string with "error consuming tunnel configuration: {err:?}"